### DISCUSSION TITLE
Is there a way to collapse (create polytomies) if branch support is below a certain value?

### DISCUSSION THREAD

**Discussion by DerekSikes** [iqtree/iqtree2#347] [Q&A]
I would like to be able to collapse branches to create polytomies for any nodes that fall below a certain value. Is this possible?

I've been trying to figure out how to do this in R with Phytools/Ape but that has been unsuccessful so am hoping I can get iQTree to do this for the tree it returns from a search.

**Comment by DerekSikes** [comment]
PS I found this -czb which collapses near zero length branches. I'd like a command that uses the UFBS value so any branches with < a user-specified value (eg, 90 or 95) are collapsed. Ideally this would also be codable based on SH-aLRT values so a branch below 80 for SH-aLRT OR below 95 for UFBS would collapse.

**Comment by roblanf** [comment]
Hi Derek,

Currently the answer is no. One thing to try is the newick utilities, which lets you do this I think.

GitHub: https://github.com/tjunier/newick_utils
Tutorial: https://gensoft.pasteur.fr/docs/newick-utils/1.6/nwutils_tutorial.pdf
Conda: https://anaconda.org/bioconda/newick_utils

Rob

**Comment by DerekSikes** [comment]
I have only quickly examined this newick utilities and the word collapse doesn't appear in the manual. I am quite surprised by this. Does Raxml allow this? Has there been a widespread movement away from caring about branch support? What is the point of generating branch support if we provide the user community with fully resolved trees, allowing them to ignore weak branches and treat them as if they were well supported? There appears to be no collapse function in phytools or Ape or any of the R packages - and perhaps this is for the same reason? They prefer to ignore weak branches. Polytomies are bad because they make the results of ACSR analyses and other comparative methods done with phylogenies even less confident than they already are. This may also be partially a result of phylogenomics, which makes virtually all branches maximally supported.. but I suspect there are still lots of weakly supported branches being published in fully resolved form (and shared newick trees that then get used by others who don't care about branch support). I'd like to see this failing of the bioinformatics community resolved through widespread agreement that weak branches should be collapsed and all software should do this by default - allowing arbitrary resolution of such nodes by user specification only. Sorry for the rant.

**Reply by DerekSikes** [reply]
This seems to allow collapsing (deletion) of weak branches: https://itol.embl.de/
